journal/modernized/02-chapter-2.adoc

```diff
@@ -399,4 +400,3 @@ to our mutual rejoicing: as did ray worthy companion her kind husband.
 On the 7th of the eighth month, 1776, Martha Winter was married, at Nottingham,
 to Richard Routh, of Manchester; and removed to his residence in that town.
 Her Journal proceeds in the next chapter.
-
```

Comment:
I see a lot of these in this PR.  Did you ever get your Atom set up to force newlines at the end of files? This seems like the opposite of what my Atom would do with these files.